### PR TITLE
UI changes 

### LINKED ISSUE #401: UI Feedback
- [x] Risk colours: Critical(Red) & High(Orange)
- [x] Vulnerability Band (remove the background and have thick left border and thin border)
- [x] Definition of CVSS on hover CVSS score
- [x] CVSS Invert colours
- [x] Upload App redesign 
- [x] Replacing Prompt with Modal box and Confirm with Confirm box

### LINKED ISSUE #441: Fix Hacky Methods
- [x] Load the data without refresh
- [x] ~Refreshing the page doesn't load the data properly~
- [x] Try using createRecord, deleteRecord everywhere 
- [x] ~Fix Select input~

### PR DESCRIPTION
**Pull request by yashwin**
fixes #401

### CODE REVIEW

File: app/app.coffee [hunk 1]
```diff
@@ -16,6 +17,7 @@ else
   config.product = ENUMS.PRODUCT.DEVKNOX
 
 installIntercom()
+surveyMonkey()
```
Comment (dhilipsiva) [inline app/app.coffee]: Man, dont install survey monjey here - link we discussed, keep a button for feedback.

File: app/components/collaboration-component.coffee [hunk 2]
```diff
@@ -31,7 +31,7 @@ CollaborationComponentComponent = Ember.Component.extend
       @get("ajax").post ENV.endpoints.collaborations, data:data
       .then (data)->
         that.send "closeModal"
-        that.get("notify").success "Collaboration will be added shortly!"
```
Comment (dhilipsiva) [inline app/components/collaboration-component.coffee]: @yashwin These things should be translated.

File: app/components/jira-project.coffee [hunk 3]
```diff
@@ -7,6 +7,23 @@ JiraProjectComponent = Ember.Component.extend
...
+  confirmCallback: ->
+    that = @
+    projectId = @get "project.id"
+    deleteJIRA = [ENV.endpoints.deleteJIRAProject, projectId].join '/'
+    @get("ajax").delete deleteJIRA
+    .then (data) ->
+      that.get("notify").success "Project has been removed"
+      that.send "closeDeleteJIRAConfirmBox"
+      setTimeout ->
+        window.location.reload() # FIXME: Hackish Way
```
Comment (dhilipsiva) [inline app/components/jira-project.coffee]: @yashwin Please create an Issue for these hacks.

Comment (yashwin) [inline app/components/jira-project.coffee]: Its here : https://github.com/appknox/irene/issues/441

File: app/app.coffee (hunk 1 above)
Comment (yashwin) [inline app/app.coffee]: ok, I will do it 👍

File: app/components/jira-project.coffee (hunk 3 above)
Comment (dhilipsiva) [inline app/components/jira-project.coffee]: @yashwin Also test cases are failing

File: app/components/collaboration-component.coffee (hunk 2 above)
Comment (yashwin) [inline app/components/collaboration-component.coffee]: @dhilipsiva : Should we do all these translations in this PR?

**Comment by dhilipsiva** [comment]
Okies. Let's create a new PR for translations. Other things are fixed right?

On August 1, 2017 12:57:41 AM GMT+05:30, Yashwin <notifications@github.com> wrote:
>yashwin commented on this pull request.
>
>
>
>> @@ -31,7 +31,7 @@ CollaborationComponentComponent =
>Ember.Component.extend
>       @get("ajax").post ENV.endpoints.collaborations, data:data
>       .then (data)->
>         that.send "closeModal"
>-        that.get("notify").success "Collaboration will be added
>shortly!"
>
>@dhilipsiva : Should we do all these translations in this PR?
>
>-- 
>You are receiving this because you were mentioned.
>Reply to this email directly or view it on GitHub:
>https://github.com/appknox/irene/pull/449#discussion_r130438909

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

File: app/components/jira-project.coffee (hunk 3 above)
Comment (yashwin) [inline app/components/jira-project.coffee]: Fixed

File: app/app.coffee (hunk 1 above)
Comment (yashwin) [inline app/app.coffee]: Done

**Comment by dhilipsiva** [comment]
@subho007 Will disable feedback for now, merge and dep[loy this. Will enable feedback back when you want to.

**Pull request merged**